lib/ping_response.rb

```diff
@@ -0,0 +1,16 @@
+class PingResponse
+  VERSION_FILE = "/.version"
+
+  UNKNOWN_VERSION_DATA_RESPONSE = {
+    version_number: "unknown",
+    build_date: nil,
+    commit_id: "unknown",
+    build_tag: "unknown"
+  }
+
+  def self.data
+    YAML.load_file(VERSION_FILE)
```

Comment:
Does the build_date field need to be ISO8601 compliant? 
If so do we rely on the .version file being written correctly, or parse and reformat?